internal/cmd/functional-benchmarker/main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/ViaQ/logerr/log"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/pkg/constants"
+	"github.com/openshift/cluster-logging-operator/test"
+	"github.com/openshift/cluster-logging-operator/test/client"
+	"github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers"
+)
+
+// HACK - This command is for development use only
+func main() {
+
+	image := flag.String("image", "quay.io/openshift/origin-logging-fluentd:latest", "The image to use to run the benchmark")
+	totalMessages := flag.Uint64("totMessages", 10000, "The number of messages to write")
+	msgSize := flag.Uint64("size", 1024, "The message size in bytes")
+	verbosity := flag.Int("verbosity", 0, "")
+	doCleanup := flag.Bool("docleanup", true, "set to false to preserve the namespace")
+	sample := flag.Bool("sample", false, "set to true to dump a sample message")
+
+	flag.Parse()
+
+	log.MustInit("functional-benchmark")
+	log.SetLogLevel(*verbosity)
+	log.V(1).Info("Args: %v", os.Args)
```

Comment:
Logger.Info() works a bit differently (https://github.com/go-logr/logr/blob/master/logr.go). This needs to be something like
```
log.V(1).Info("Starting functional benchmarker", "command line", os.Args)
```
which will log
```
{"command line":["./bin/functional-benchmarker","-verbosity=9"],"component":"functional-benchmark","message":"Starting functional benchmarker","ts":"2020-12-02T19:44:00.546444871Z"}
```

The way it is now logs
```
{"component":"functional-benchmark","message":"Args: %v","ts":"2020-12-02T20:14:02.68607955Z"}
```